src/lib/booleans.js

```diff
@@ -0,0 +1,118 @@
+function negate(a) {
+  return !a;
+}
+
+function both(a, b) {
+  return a && b;
+}
+
+function either(a, b) {
+  return a || b;
+}
+
+function none(a, b) {
+  return !a && !b;
+}
+
+function one(a, b) {
+  if (!a && !b) {
+    return false;
+  }
+  return !a || !b;
```

Comment:
You can rewrite this function so it will return false based on one condition, and return true if not:
```
  if (!a && !b) {
    return false;
  }
  return true;
```
Going a bit further you could turn it into a simple ternary operator:
```
return (!a && !b) ? false : true;
```
Which as it happens my editor simplified to just this!
```
  return !(!a && !b);
```




```
  if (!a && !b) {
    return false;
  }
  return !a || !b;
```